scripts/pi-hole/js/network.js

```diff
@@ -154,9 +159,7 @@ $(function () {
       }
 
       $("td:eq(0)", row).html(ips.join("<br>"));
-      $("td:eq(0)", row).on("hover", function () {
-        this.title = data.ips.join("\n");
-      });
+      $("td:eq(0)", row).attr("title", iptxt.join("\n"));
```

Comment:
Maybe we could put this line inside the `if (data.ips.length > MAXIPDISPLAY)` block, to only display the title/tooltip when there are more than 3 items.

Reply:
I agree. There is not much sense to show the hover with exactly the same information as displayed in the cell.

Reply:
Done (v6 and v5 PRs).